doc/input-output-reference/src/overview/group-zone-forced-air-units.tex

```diff
@@ -3899,13 +3906,13 @@ \subsubsection{Inputs}\label{inputs-14-018}
 
 This alpha field defines the name of the terminal unit's supply air fan.
 
-\paragraph{Field: Outside Air Mixer Object Type}\label{field-outside-air-mixer-object-type}
+\paragraph{Field: Outdoor Air Mixer Object Type}\label{field-outdoor-air-mixer-object-type}
 
-This alpha field contains the identifying type of outside air mixer specified for the terminal unit. Outside air mixer type must be \textbf{OutsideAir:Mixer}. This field should be left blank when an outside air mixer is not simulated.
+This alpha field contains the identifying type of outdoor air mixer specified for the terminal unit. Outdoor air mixer type must be \textbf{OutdoorAir:Mixer}. This input field should be left blank when the VRF terminal unit is placed in \textbf{AirTerminal:SingleDuct:Mixer} object. If this field is left blank, an outdoor air mixer is not simulated.
```

Comment:
@Nigusse For consistency with other similar text, "AirTerminal:SingleDuct:Mixer" should be italics here instead of bold.  Same in line 3915 below.